cylc/flow/network/authentication.py

```diff
@@ -13,59 +13,19 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
-"""Network authentication layer."""
+"""Standard encode and decode methods for the network authentication layer."""
 
 import getpass
+import json
 
-from jose import jwt
 
-from cylc.flow.suite_files import SuiteFiles, get_auth_item
+def encode_(message):
+    """ Encode a message from JSON format to a string. """
```

Comment:
I'm the names of these methods and the docstrings a little hard to understand - had to go check the way json works:
propose either:
> Encode a message in a dict as a JSON string.
or
```python
args:
    message (dict):
        message to encode as a JSON string
returns (str):
    A string containing a JSON representative of the message.
```

Similarly below I think that 
> Load a JSON formatted string into a dict.

Perhaps `encode_` and `decode_` are unhelpful names and could be, say `json_to_dict` and `dict_to_json`?

Reply:
Thanks for raising this. I will leave the names as they are since they are a concise & distinct description of the function purposes, so I think are fitting, but I've amended the docstrings to hopefully make the nature clear to those not so familiar with the terms.